qcengine/programs/cfour/harvester.py

```diff
@@ -227,6 +229,8 @@ def harvest_outfile_pass(outtext):
         psivar["MP3 TOTAL ENERGY"] = mobj.group(8)
         psivar["MP2.5 CORRELATION ENERGY"] = dmp2 + Decimal("0.500000000000") * dmp3
         psivar["MP2.5 TOTAL ENERGY"] = psivar["MP2.5 CORRELATION ENERGY"] + psivar["SCF TOTAL ENERGY"]
+        psivar["MP3 SINGLES ENERGY"] = Decimal(mobj.group(1)) + Decimal(mobj.group(5))
```

Comment:
To my knowledge, we don't have CI or tests exercising these codepaths. Do we accept this for now?

Reply:
yeah, I've tested them over at qcdb. at that was only backup testing for psi4 (which only has one open-shell implementation of mp3). afraid testing is trickle-down to qcng from other projects.

Reply:
Understood, thank you! This clarifies it for me as well. Appreciated!